www/i18n/en-US.js

```diff
@@ -1099,6 +1099,7 @@ export default {
   "playedXTimes:other": "Played %s times",
   "coordinates": "Coordinates",
   "coordinateTraining": "Coordinate training",
+  "previousScore": "Previous score: %s",
```

Comment:
Please don't modify these files: they're auto generated. Just put an english string for now where you need it, we'll consider translating it later.

Reply:
> Please don't modify these files: they're auto generated. Just put an english string for now where you need it, we'll consider translating it later.

Apologies, I didn't realize they were generated, but they are reverted now.

I'm interested to learn how that's accomplished.